modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/internal/display/context/ItemSelectorRepositoryEntryManagementToolbarDisplayContext.java

```diff
@@ -95,6 +100,22 @@ public CreationMenu getCreationMenu() {
 			DLPortletToolbarContributorRegistryUtil.
 				getDLPortletToolbarContributor();
 
+		long folderId = GetterUtil.getLong(
+			_httpServletRequest.getAttribute(
+				"liferay-item-selector:repository-entry-browser:folderId"));
+
+		Folder folder = null;
+
+		try {
+			folder = DLAppLocalServiceUtil.getFolder(folderId);
+		}
+		catch (PortalException portalException) {
+			_log.error(portalException);
+		}
+
+		_liferayPortletRequest.setAttribute(
+			WebKeys.DOCUMENT_LIBRARY_FOLDER, folder);
+
```

Comment:
Hi @quanhuynhces , you can add a private method to get `folderId` like this:
```
private String _getFolderId() {
		 PortletURL uploadURL = (PortletURL) 
			_httpServletRequest.getAttribute(
				"liferay-item-selector:repository-entry-browser:uploadURL");
		 return ((LiferayPortletURL)uploadURL).getParameter("folderId");
}
```

Reply:
@TaiPhamLR, thank you. I've updated the code follow your comment and I think we should create _getFolder() method for cleaner.

Reply:
Thank you @TaiPhamLR. Following your comment but I change a little bit to getFolder() method. Is this more suitable, please let me know.